docs/docs/Integrations/Google/integrations-setup-google-oauth-langflow.md

```diff
@@ -0,0 +1,101 @@
+4. To navigate to the OAuth consent screen, go to **APIs & Services** and click **OAuth consent screen**.
+5. Set up your OAuth consent screen with essential app details like the application name, user support email, required [scopes](https://developers.google.com/identity/protocols/oauth2/scopes) (permissions your app needs), and authorized domains.
+
+![OAuth Consent Screen](/img/google/setup-oauth-consent-screen.png)
+
+6. Ensure you **publish** your app if it’s not restricted to internal use.
+7. To create an **OAuth Client ID**, navigate to **Credentials** and select **Create Credentials** > **OAuth Client ID**.
+8. Choose **Desktop app** as the application type.
+
+9. Click **Create**. A Client ID and Client Secret are created. Download the credentials as a JSON file to your local machine and save it securely.
+
+![OAuth Client ID and Secret](/img/google/create-oauth-client-id.png)
+
+---
+
+## Retrieve Access and Refresh Tokens
+
+With your OAuth application configured and your credentials JSON file created, follow these steps to authenticate the Langflow application.
+
+1. Create a new project in Langflow.
+2. Add a **Google OAuth Token** component to your flow.
+3. Input in the field **Credentials File** on the Google OAuth Token component, the JSON file containing the Client ID credentials you downloaded from Google in the [previous steps](#5b8981b15d86192d17b0e5725c1f95e7).
+4. Click the **Play** button in the **Google OAuth Token** component to authenticate your application.
+When the component code is executed, a new tab may open in the browser to authenticate your application using your Google Cloud account. You must authenitcate with the account where you created the OAuth credentials for your application.
+If a new tab does not open automatically, check the Langflow **Logs** for the Google authentication URL. Open this URL in your browser to complete the authentication. Only after authenticating will the JSON token be generated.
+
+5. After successful authentication, your Langflow application can now request and refresh tokens for your application. These tokens enable Langflow to interact with Google services on your behalf and execute the requests you’ve specified.
+By default, token validity is managed by Google’s servers. In Langflow, tokens refresh automatically after initial authentication. However, if your application is inactive for an extended period, the tokens may expire, requiring you to re-authenticate to resume use in Langflow.
+
+## Configure Google Components in Langflow
+
+In this example, use the **Google Drive Loader** component to load a text file hosted on Google Drive, translate the text to Spanish, and return it to a chat output.
+
+For a pre-built example of this flow, download this JSON file to your local machine.
+
+- Flow Google Drive Docs Translations Example -
+  (<a href="./files/Google_Drive_Docs_Translations_Example.json" download>Download link</a>)
```

Comment:
```suggestion
```

Reply:
Hi @mendonk Could you check the changes? https://github.com/langflow-ai/langflow/pull/4382/commits/0ae39d163900b731433691ab45cdbdeb3676cd24